packages/valory/skills/decision_maker_abci/behaviours/decision_receive.py

```diff
@@ -277,7 +277,11 @@ def _get_mocked_bet(self) -> Bet:
         liquidity_amounts = self.shared_state.liquidity_amounts
         liquidity_prices = self.shared_state.liquidity_prices
         markets = list(liquidity_amounts.keys())
-        question_id = self.shared_state.mock_data.id
+        if self.shared_state.mock_data is not None:
+            question_id = self.shared_state.mock_data.id
+        else:
+            raise ValueError("No mocked data information")
```

Comment:
This is duplicated below, you may create a property for safely accessing the `self.shared_state.mock_data`.

Reply:
Without the BenchmarkingMockData there is no benchmarking mode so I guess this use-case is already covered in the code even before. No extra checks needed here.